docker-hub/vulnerability-scanning.md

```diff
@@ -4,7 +4,11 @@ keywords: scanning, vulnerabilities, Hub, Snyk
 title: Hub Vulnerability Scanning
 ---
 
-> Docker Hub Vulnerability Scanning is available for developers subscribed to a Pro or a Team plan. For more information about the pricing plans, see [Docker Pricing](https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade){: target="_blank" rel="noopener" class="_"}.
+{% include upgrade-cta.html
+  body="Docker Hub Vulnerability Scanning is available for developers subscribed to a Pro or a Team plan. Upgrade your existing account to start using Hub Vulnerability Scanning."
```

Comment:
Upgrade _plan_, not account. Reduce repetition.

```suggestion
  body="Docker Hub Vulnerability Scanning is available for developers subscribed to a Pro or a Team plan. Upgrade your existing plan to get automatic vulnerability scans for your images."
```

Reply:
Updated